### ISSUE TITLE
Wrong SQL validation

### ISSUE THREAD

**Issue by juanignaciosl** [CartoDB/carto-vl#497]
This is valid for CARTO VL:

```js
    const source = new carto.source.SQL(`select
    cartodb_id, ST_Simplify(the_geom, 10000) the_geom, ST_Simplify(the_geom_webmercator, 10000) the_geom_webmercator,fpost8
    from ew_mortality_children`)
```

... but this (a new line for the last column) is not:

```js
    const source = new carto.source.SQL(`select
    cartodb_id, ST_Simplify(the_geom, 10000) the_geom, ST_Simplify(the_geom_webmercator, 10000) the_geom_webmercator,
    fpost8
    from ew_mortality_children`)
```

**Comment by jgoizueta** [comment]
Are you source you're using a recent version of Carto VL? I think this was fixed in #441

**Comment by juanignaciosl** [comment]
I was using 0.3.0 (the one mentioned in the documentation). It doesn't happen in 0.4.0 👍
